Module_09/ex01/README.md

```
@@ -0,0 +1,99 @@
+
+### Example
+```
+./RPN "21 21 +"
+```
+
+This will calculate 21 + 21 and output the result (42).
+
+## Supported Operations
+
+The calculator supports the following operations:
+- Addition (`+`)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)